package.json

```diff
@@ -1,5 +1,7 @@
 {
   "name": "alinka-website",
+  "homepage": "https://alinka.io",
```

Comment:
These changes were probably introduced in different pull request previously and this diff is a result of wrong back merge.